### PR TITLE
HKButton

### PR DESCRIPTION
**Pull request by caseywatts**
Rename `Button` to `HKButton` for naming consistency.

### CODE REVIEW

File: stories/HKButton.stories.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 import { storiesOf } from '@storybook/react'
 import { action } from '@storybook/addon-actions'
-import { Button } from '../src'
+import { HKButton } from '../src'
```
Comment (uhoh-itsmaciek) [inline stories/HKButton.stories.js]: Can you also change the `Button` in `storiesOf()` [here](https://github.com/heroku/react-hk-components/pull/10/files#diff-27de31c5a74678afb519adfe8d94ab16R12)?

Comment (caseywatts) [reply stories/HKButton.stories.js]: done!

Comment (uhoh-itsmaciek) [reply stories/HKButton.stories.js]: And pushed? I don't see your changes here or in the review app...

Comment (caseywatts) [reply stories/HKButton.stories.js]: oop - now it is

**Pull request merged**